web/src/lib/components/faces-page/person-side-panel.svelte

```diff
@@ -137,6 +164,28 @@
           }
         }
 
+        for (const [id, face] of Object.entries(selectedPersonToAdd)) {
+          if (face.person) {
+            await reassignFacesById({
+              id: face.person.id,
+              faceDto: { id },
+            });
+          } else {
+            const data = await createPerson({ personCreateDto: {} });
+            peopleToCreate.push(data.id);
+            await reassignFacesById({
+              id: data.id,
+              faceDto: { id },
+            });
+          }
+        }
+
+        for (const [id, _] of Object.entries(selectedFaceToRemove)) {
```

Comment:
```suggestion
        for (const [id] of Object.entries(selectedFaceToRemove)) {
```
<img width="246" alt="Screenshot 2024-05-11 at 21 34 46" src="https://github.com/immich-app/immich/assets/36593685/34b81033-a225-4cf0-972f-2a1ae3ea709e">